prombench/.gitignore

```diff
@@ -1,5 +1 @@
 /prombench
```

Comment:
Is the gitignore really working from a subdir?

Reply:
Yes, it's working.